apps/fluent-tester/src/FluentTester/theme/applyTheme.ts

```diff
@@ -0,0 +1,78 @@
+import { OfficePalette, PartialTheme, Theme } from '@fluentui-react-native/framework';
+import { createPartialOfficeTheme, getThemingModule } from '@fluentui-react-native/win32-theme';
+import { testerTheme } from './CustomThemes';
+
+export type ThemeNames = 'Fluent' | 'Office' | 'Caterpillar';
+
+export const themeChoices = [
+  { label: 'Fluent', value: 'Fluent' },
+  { label: 'Office', value: 'Office' },
+  { label: 'Caterpillar', value: 'Caterpillar' },
+];
+
+export function applyCaterpillarTheme(parent: Theme): PartialTheme {
```

Comment:
I saw you mention this in the description. What is a Caterpillar Theme?

Reply:
This is just example code and is non-shipping. The idea (taken from Fluent) is for a company like Caterpillar (I believe a real Fluent/Fabric customer) to be able to inject their brand into the theming.

Keep in mind all of these things were already present in the tester, this just reorganizes them.